src/models/conUOpenDataCourse.model.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const openDataCourseSchema = new Schema({
+    ID: {type: String, required: true},
```

Comment:
Change this to courseID to make it consistent with the other models

Reply:
I'm using ID to match the record fields obtained from ConU open data. While I can change the field name, it will not be reflected for batch inserts created using .insertMany(...) due to the way it works. It will be reflected for records created using .save(...) however. 

It is doable with some data manipulation of Open Data records before batch inserts into our database, although it will be extra processing before database insertion. Let me know if that would be okay, or if you have a more efficient way of doing it.

Reply:
I see, leave it as it is then.